operator/webhook/introute/sync.py

```diff
@@ -211,14 +284,14 @@ def _create_pod_template(parent, labels, integration_image):
                     "livenessProbe": {
                         "httpGet": {
                             "path": "/actuator/health/liveness",
-                            "port": 8080,
+                            "port": 80,
```

Comment:
:question: Why the switch to `80`? Just to use the standard port number?

Reply:
I ended up setting it to 8443 or 8080 depending on whether TLS was used. I believe to use 80 or 443 it would have to run in a privileged container, so 80 wouldn't work.